app/src/main/java/com/example/paulsuarez/a28_filterable_lists/models/Data.java

```diff
@@ -0,0 +1,42 @@
+package com.example.paulsuarez.a28_filterable_lists.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Data {
```

Comment:
Similar to the Constructor class, the name of this one should be more descriptive about specifically what data. Especially since you generally have more than one model class in an application, so if you call the class "Data", you don't really have any good options for naming a second model class. :)